src/cat.rs

```diff
@@ -154,41 +148,24 @@ fn show_locals<W: io::Write>(locals: &[String], _opts: &Options, output: &mut W)
         .filter(|&name| {
             name != "globals.local" && name != "pre-globals.local" && name != "post-globals.local"
         })
-        .filter_map(|name| match find_profile(name) {
-            Some(path) => Some(path),
-            None => {
-                warn!("{} could not be found.", name);
-                None
-            }
+        .filter_map(|name| {
+            let profile_flags = ProfileFlags::default_with(ProfileFlags::READ);
+            Profile::new(name, profile_flags).ok()
```

Comment:
This will be silent on errors. IMHO a message should be printed if something goes wrong.

Reply:
Noted.